.github/workflows/autodocs-images.yaml

```diff
@@ -0,0 +1,54 @@
+name: Build Images Reference Docs
+on:
+  workflow_dispatch:
+env:
+  WORKDIR: images
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: "Check out Destination Repo"
+        uses: actions/checkout@v2
```

Comment:
```suggestion
        uses: actions/checkout@755da8c3cf115ac066823e79a1e1788f8940201b # v3.2.0
```

Reply:
I think there's a preference for using commit hashes with actions, since tags can change invisibly from underneath us. Any reason to use v2 over v3 of this action?

Reply:
Ah, thanks! I  have been following mostly the examples they have and I didn't know about using hashes, makes sense. And no reason to use v2, it was just because I copied from another action I had :) thanks!